review-me/review-me.js

```diff
@@ -0,0 +1,26 @@
+// hello.js
+
```

Comment:
2  :  Inline  comment  does  not  follow  J SD oc  or  structured  format  -  style